kvbc/include/sparse_merkle/base_types.h

```diff
@@ -66,7 +66,7 @@ class CustomPrefix {
   CustomPrefix() = default;
   CustomPrefix(size_t size) {
     ConcordAssert(size <= SIZE_IN_BYTES);
-    prefix_.assign('\0', size);
+    prefix_.assign(size, '\0');
```

Comment:
how could it pass before ?

Reply:
The unit test `tree_test.cpp` stores its database in RAM and does not serialize the elements. The elements of the tree are:
`  map<std::string, Version> latest_version_;
  map<LeafKey, LeafNode> leaf_nodes_;
  map<InternalNodeKey, BatchedInternalNode> internal_nodes_;`
[test_db.h:43](https://github.com/vmware/concord-bft/blob/66b249a1e709ed4d1baf5f4e5e73bae19bbadd58/kvbc/test/sparse_merkle/test_db.h#L43)
In the more advanced test we are planning we will store the test DB's elements serialized in a way similar to:
`map<std::string, Version> latest_version_;
map<std::string, std::string> nodes_;`